odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -687,14 +701,29 @@ PROCEDURE READCONFIGFILESETTINGS
         DO
          COLONPOS=CHARACTERSEARCH(PARAMVALUE,":")
          IF COLONPOS>1 AND
-            COLONPOS<LENGTH(PARAMVALUE) AND
-            PARAMOTACOUNT<PARAMOTAMAX THEN
+            COLONPOS<LENGTH(PARAMVALUE) THEN
           DO
-           PARAMOTACOUNT=PARAMOTACOUNT+1
-           PARAMOTASHARETYPE(PARAMOTACOUNT)=VALUE(SEGMENT(PARAMVALUE,1,COLONPOS-1))
-           PARAMOTAAMOUNT(PARAMOTACOUNT)=MONEY(VALUE(SEGMENT(PARAMVALUE,COLONPOS+1,LENGTH(PARAMVALUE))))
-          END
-        END
+           IF UPPERCASE(SEGMENT(PARAMVALUE,1,COLONPOS-1))="XXXX" THEN
+            DO
+             DEFAULTOTAVALUE=MONEY(VALUE(SEGMENT(PARAMVALUE,COLONPOS+1,LENGTH(PARAMVALUE))))
+             TMPCHR=FORMAT("###,###,##9.99",DEFAULTOTAVALUE)
+             CALL NLS
+             DEFAULTOTAVALUECHR=TMPCHR
+             BNODEBUGMSG="Default OTA value FOUND: $"+DEFAULTOTAVALUECHR
+             CALL BNODEBUGMSGADD
+            END
+           ELSE
```

Comment:
Could just do:
```
ELSE IF PARAMOTACOUNT<PARAMOTAMAX THEN
```
to get rid of a couple lines of code.

Reply:
@rrobison44 changed.